main/settings_test.py

```diff
@@ -113,6 +125,24 @@ def test_csrf_trusted_origins(settings_sandbox):
     ]
 
 
+def test_mitol_apigateway_allowed_redirect_hosts(settings_sandbox):
+    """Verify that we can configure MITOL_APIGATEWAY_ALLOWED_REDIRECT_HOSTS with a var"""
+    # Test the default
+    settings_vars = settings_sandbox.get()
+    assert settings_vars.get("MITOL_APIGATEWAY_ALLOWED_REDIRECT_HOSTS") == []
```

Comment:
Test fails for this - probably need to add this to the `settings_sandbox` up above (or have it check for the default)